myriad-scheduler/src/main/java/org/apache/myriad/scheduler/event/handlers/ResourceOffersEventHandler.java

```diff
@@ -243,7 +243,7 @@ private boolean meetsConstraint(Offer offer, Constraint constraint) {
 
   private void checkResource(boolean fail, String resource) {
     if (fail) {
-      LOGGER.info("No " + resource + " resources present");
+      LOGGER.error("No " + resource + " resources present");
```

Comment:
Cool, will set to debug

On Wed, Jul 13, 2016 at 8:50 AM, Darin <notifications@github.com> wrote:

> In
> myriad-scheduler/src/main/java/org/apache/myriad/scheduler/event/handlers/ResourceOffersEventHandler.java
> <https://github.com/apache/incubator-myriad/pull/84#discussion_r70618298>:
>
> > @@ -243,7 +243,7 @@ private boolean meetsConstraint(Offer offer, Constraint constraint) {
> >
> >    private void checkResource(boolean fail, String resource) {
> >      if (fail) {
> > -      LOGGER.info("No " + resource + " resources present");
> > +      LOGGER.error("No " + resource + " resources present");
>
> Again not an error, it's possible to have an offer with 0.0 cpu resources,
> but with 2048 mem resources. While it's not useful to the framework, it's
> not an error. Debug or info.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/incubator-myriad/pull/84/files/b4fbad0582bb71b35968b1cdfb4e85e629ec3758#r70618298>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKSRgUr4xKdd8bc2LRVa3Bo-T4N6M1T3ks5qVN8mgaJpZM4JLUAy>
> .
>